miniscope_io/models/devupdate.py

```diff
@@ -0,0 +1,123 @@
+"""
+Models for device update commands.
+"""
+
+from enum import Enum
+
+import serial.tools.list_ports
+from pydantic import BaseModel, ConfigDict, field_validator, model_validator
+
+
+class DeviceCommand(Enum):
+    """Commands for device."""
+
+    REBOOT = 200
+
+
+class UpdateCommandDefinitions:
+    """
+    Definitions of Bit masks and headers for remote update commands.
+    """
+
+    # Header to indicate target/value.
+    # It probably won't be used in other places so defined here.
+    id_header = 0b00000000
+    target_header = 0b11000000
+    LSB_header = 0b01000000
+    MSB_header = 0b10000000
+    LSB_value_mask = 0b000000111111  # value below 12-bit
+    MSB_value_mask = 0b111111000000  # value below 12-bit
+    reset_byte = 0b11111111
+
+
+class UpdateTarget(Enum):
```

Comment:
```suggestion
class UpdateTarget(int, Enum):
```

if you do this, then the members of the enum behave like integers as well as enum members:

```python
class RegularEnum(Enum):
    x = 1
    y = 2
    z = 3
    
class IntEnum(int, Enum):
    x = 1
    y = 2
    z = 3
```

```python
>>> isinstance(RegularEnum.x, int)
False
>>> RegularEnum.x + 5
Traceback (most recent call last):
  File "<input>", line 1, in <module>
TypeError: unsupported operand type(s) for +: 'RegularEnum' and 'int'
>>> RegularEnum.x in RegularEnum
True
>>>
>>> isinstance(IntEnum.x, int)
True
>>> IntEnum.x + 5
6
>>> IntEnum.x in IntEnum
True
```

Reply:
Oh this is nice

Reply:
done